hudi-spark/src/test/java/DataSourceUtilsTest.java

```diff
@@ -17,18 +17,58 @@
  */
 
 import org.apache.hudi.DataSourceUtils;
+import org.apache.hudi.DataSourceWriteOptions;
+import org.apache.hudi.client.HoodieWriteClient;
+import org.apache.hudi.common.model.HoodieRecord;
+import org.apache.hudi.common.util.Option;
+import org.apache.hudi.config.HoodieWriteConfig;
+import org.apache.hudi.table.NoOpBulkInsertPartitioner;
 
 import org.apache.avro.Schema;
 import org.apache.avro.generic.GenericData;
 import org.apache.avro.generic.GenericRecord;
-import org.junit.jupiter.api.Test;
+import org.apache.spark.api.java.JavaRDD;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
```

Comment:
@kwondw We're moving to Junit 5.. could you please switch to Junit 5 api ?  thank you.

Reply:
I'm using mockito, until mockito upgrade to version that support junit 5, it doesn't seem to work.
I think same as https://github.com/apache/incubator-hudi/blob/master/hudi-common/src/test/java/org/apache/hudi/common/table/view/TestPriorityBasedFileSystemView.java#L54

Reply:
@kwondw my bad...i should have migrated mockito together when i added the junit 5 dep.. I've opened up #1521 to migrate mockito.
@vinothchandar @yanghua To make all ongoing PRs ready to use JUnit 5, we would hope to get  #1521 merged sooner. Thank you.

Reply:
Thanks for @yanghua prompt review.. the mockito lib was upgraded. it should work for junit 5 now..